collector/corosync/parser.go

```diff
@@ -96,7 +96,11 @@ func parseRingIdAndSeq(cfgToolOutput []byte) (string, uint64, error) {
 	nodeRe := regexp.MustCompile(`(?m)Ring ID:\s+(\w+)/(\d+)`)
 	matches := nodeRe.FindSubmatch(cfgToolOutput)
 	if matches == nil {
-		return "", 0, errors.New("could not find Ring ID line")
+		nodeRe = regexp.MustCompile(`(?m)Ring ID:\s+(\w+).(\d+)`)
```

Comment:
since there is so little changing, maybe we could reuse the same regex instead of adding an additional if block?

`(?m)Ring ID:\s+(\w+).|/(\d+)` should do the trick.

Reply:
sure!

Reply:
done